src/lib/project.js

```diff
@@ -1,57 +1,159 @@
-const chalk = require('chalk');
 const fs = require('fs-extra');
 const path = require('path');
 const ora = require('ora');
 const sh = require('shelljs');
 const util = require('util');
 const gittar = require('gittar');
+const { prompt } = require('enquirer');
+const { spawnSync } = require('child_process');
+const { red, green, reset } = require('chalk');
 
-const _green = chalk.green;
-const _red = chalk.red;
 const shExec = util.promisify(sh.exec);
 
 /**
  * Create a new zkApp project with recommended dir structure, Prettier config,
  * testing lib, etc. Warns if already exists and does NOT overwrite.
  * @param {string} name  Desired dir name or path. Will recursively create
  *                       dirs without overwriting existing content, if needed.
- * @return {Promise<void>}
+ * @return {promise<void>}
```

Comment:
Shouldn't this be `Promise`?

Intellisense is giving me this currently:
`type promise = /*unresolved*/ any`

Reply:
Are you referring to the upper vs lower casing? 
Intention was to use the same lowercase as all other types currently have.
Is this ESLint feedack you're looking at? I'm interested to get set up similarly

Reply:
I don't believe it's ESLint, I just opened up a web editor and hovered over the types. I opened a web editor by pressing `.` in this pull request!

Reply:
Interesting re: the editor!

If your suggestion is to use `Promise` with a capital first letter, would it be fair to interpret your suggestion to change all types to use capital letter? E.g. String, Boolean, etc. Otherwise it'll be inconsistent, no? 

I've always seen and used lowercase (similar to https://jsdoc.app/about-getting-started.html), but I can see your logic too. Whichever way we decide to go is less important than consistency imo.

Reply:
I just intend to change `Promise` to a capital letter first, since that's the correct type for promises. The primitive types like `string`, `boolean`, etc are fine to leave as lowercase and I would actually prefer to leave them as is. There is a `string` primitive and an object wrapper `String` but this is not the case for `promise` vs `Promise`.